### PR TITLE
Reconcile CDI node placement

### PR DESCRIPTION
**Pull request by nunnatsa**
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

**Release note**:

```release-note
Reconcile CDI node placement configurations by copying HCO node placement configurations
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: pkg/controller/hyperconverged/hyperconverged_controller_components_test.go [hunk 1]
```diff
@@ -552,7 +552,113 @@ var _ = Describe("HyperConverged Components", func() {
 			Expect(*foundResource.Spec.UninstallStrategy).To(Equal(*expectedResource.Spec.UninstallStrategy))
 		})
 
-		// TODO: add tests to ensure that HCO properly propagates NodePlacement from its CR
+		It("should add node placement if missing in CNAO", func() {
```
Comment (tiraboschi) [inline pkg/controller/hyperconverged/hyperconverged_controller_components_test.go:555]: "if missing in CDI"

**Comment by orenc1** [comment]
the bot should override index image azure
/override-bot

**Comment by orenc1** [comment]
@nunnatsa please rebase to include the upgrade test fix merged in #815

**Comment by tiraboschi** [comment]
hco-e2e-image-index-gcp lane succeeded.
/override ci/prow/hco-e2e-image-index-azure
hco-e2e-image-index-gcp lane succeeded.
/override ci/prow/hco-e2e-image-index-aws
hco-e2e-upgrade-prev-aws lane succeeded.
/override ci/prow/hco-e2e-upgrade-prev-azure

**Review by tiraboschi — APPROVED** [review]

**Pull request merged**